cmake/LLVMHelper.cmake

```diff
@@ -1,15 +1,17 @@
 # =============================================================================
 # LLVM/Clang needs to be linked with either libc++ or libstdc++
 # =============================================================================
+
+find_package(LLVM REQUIRED CONFIG)
+
+# include LLVM header and core library
+llvm_map_components_to_libnames(LLVM_LIBS_TO_LINK core)
+set(CMAKE_REQUIRED_INCLUDES ${LLVM_INCLUDE_DIRS})
+set(CMAKE_REQUIRED_LIBRARIES ${LLVM_LIBS_TO_LINK})
+
```

Comment:
Just FYI : I changed this because this was a bug - we were finding LLVM only for Clang compiler.